src/app/lib/utilities/ichefURL/index.js

```diff
@@ -2,28 +2,27 @@
 const WEBP_ORIGIN_CODES = ['cpsdevpb', 'cpsprodpb'];
 
 const buildPlaceholderSrc = (src, resolution) => {
-  const parts = src.split('/');
-  const [domain, media, imgService, width, ...extraParts] = parts;
-  const definedWidth = width.replace('$width', resolution);
-  const domainWithProtocol = `https://${domain}`;
-
+  // return urn schemes unmodified
+  if (src.includes('urn:')) return src;
+  const noProtocolSrc = src.replace('https://', '');
+  const urlParts = noProtocolSrc.split('/');
+  const [domain, mediaType, imgService, ...extraParts] = urlParts;
+  // Slice use to cut off the old resolution
+  const newExtraParts = extraParts.slice(1);
```

Comment:
Could maybe make this comment unnecessary

```suggestion
  const urlPartsWithoutResolution = extraParts.slice(1);
```

Reply:
or `remainingUrlPartsWithoutResolution` nothing wrong with a long var name!